crossfire/__init__.py

```diff
@@ -51,16 +61,42 @@ def occurrences(
     )
 
 
+def _flatten_df(row, column_name):
+    column_data = row[column_name]
+    return Series(
+        {f"{column_name}_{key}": value for key, value in column_data.items()}
+    )
+
+
+def is_empty(data):
+    if HAS_PANDAS and isinstance(data, DataFrame):
+        return data.empty
+    else:
+        return not data
```

Comment:
What is the role of this `else` if the `if` already returns?

```suggestion
    return not data
```